glusterd2/main.go

```diff
@@ -119,6 +121,29 @@ func main() {
 		log.WithError(err).Fatal("Failed to generate local auth token")
 	}
 
+	// Get the Jaeger endpoints from config file
+	jaegerEndpoint := config.GetString("jaegerendpoint")
+	jaegerAgentEndpoint := config.GetString("jaegeragentendpoint")
+
+	// Create the Opencensus Jaeger exporter endpoints to log traces and spans
+	exporter, err := jaeger.NewExporter(jaeger.Options{
```

Comment:
Have this be a function `InitTracing` that returns an exporter. The GD2 main function will call this function, and then call `defer exporter.Flush()` if a non-nil return happens.

Reply:
The exporter initialization is moved to tracing.InitJaegerExporter() which is specific to Jaeger. If we support tracing using some other mechanism/endpoint(s) in the future, a new function can be defined as needed.